cmake/FindMbedTLS.cmake

```diff
@@ -0,0 +1,17 @@
+find_path(MBEDTLS_INCLUDE_DIRS mbedtls/ssl.h)
+
+# mbedtls-3.0 changed headers files, and we need to ifdef'out a few things
+find_path(MBEDTLS_VERSION_GREATER_THAN_3 mbedtls/build_info.h)
```

Comment:
is this variable technically `MBEDTLS_VERSION_3_OR_GREATER`?

Reply:
maybe, we are not relying on it though. I 'borrowed' this file from another project